connectors/tests/test_es_client.py

```diff
@@ -79,12 +79,18 @@ async def test_es_client_auth_error(mock_responses, patch_logger):
 
 
 @pytest.mark.asyncio
-async def test_es_client_no_server(patch_logger):
+async def test_es_client_no_server(mock_responses, patch_logger):
```

Comment:
This is a different test. you have a server "answering" if you mock responses.

Reply:
you can trigger an `elasticsearch.ConnectionError` when `client.info` is called to simulate the same issue

Reply:
Good call, done!